examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -268,35 +333,46 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
         n_cells += cells_per_event.total_size();
         n_measurements += measurements_per_event.total_size();
         n_spacepoints += spacepoints_per_event.total_size();
+        n_spacepoints_cuda += spacepoints_per_event_cuda.total_size();
         n_seeds_cuda += seeds_cuda.size();
```

Comment:
Thanks for adding the change as requested, however seeds_cuda will be empty if `run_cpu=0` right? And the statistics wouldn't make sense in that case, is that okay?

Reply:
Yeah you are right. I don't see a nice way to handle the statistics part without transferring the memory to host :/
We might need to change statistics part more ergonomically at some point.